src/Our.Umbraco.Ditto/ComponentModel/ValueResolvers/AppSettingValueResolver.cs

```diff
@@ -28,12 +28,21 @@ public class AppSettingValueResolver : DittoValueResolver<AppSettingAttribute>
         /// </returns>
         public override object ResolveValue(ITypeDescriptorContext context, AppSettingAttribute attribute, CultureInfo culture)
         {
-            if (string.IsNullOrWhiteSpace(attribute.AppSettingKey))
+            // default to attribute value
+            string appSettingKey = attribute.AppSettingKey;
```

Comment:
I think `PropertyDescriptor` is never null, (but I could be wrong? @mattbrailsford?).

On another note, we could get this into a shorter statement, as we do in [`UmbracoPropertyValueResolver.cs` line 33](https://github.com/leekelleher/umbraco-ditto/blob/develop/src/Our.Umbraco.Ditto/ComponentModel/ValueResolvers/UmbracoPropertyValueResolver.cs#L33)

    var umbracoPropertyName = attribute.PropertyName ?? (context.PropertyDescriptor != null ? context.PropertyDescriptor.Name : string.Empty);

*(not that we're aiming for crazy single-lines-of-code)*